evaluation/welcome_text.md

```diff
@@ -0,0 +1,29 @@
+```
+Dear [NAME],
+
+thank you very much for participating in our small evaluation for the Master Team Project.
+
+You will find detaield instructions concerning your task here: https://d067659.github.io/tpmc_documentation/evaluation/use_case_based_evaluation_process.html.
+
+For now, please complete the tasks 1 to 4 (Creating a routine and perform Create, Update, Delete operations on functions).
+
+To gain a better understanding, it is recommendable to read through the documentation: https://d067659.github.io/tpmc_documentation/technical_documentation/overall_microservice_archtitecture.html
+Especially the Frontend Documentation will provide better insights on the task you are about to complete: https://d067659.github.io/tpmc_documentation/technical_documentation/frontend_service.html
```

Comment:
I would normally say you don't have to read a technical documentation to understand how a software works. In this case it maybe makes sense to understand the rough architecture but I would rather say that they should read some kind of user guide (which I believe we don't have at the moment.

Reply:
yes, a user guide would be very helpful!